content/components/relative-time.mdx

```diff
@@ -1,16 +1,24 @@
 ---
-title: Relative time
+title: Dates & Times
```

Comment:
I'm tempted to rename this so it is clear that we expect all dates & times to use this format side wide. Thoughts?

Reply:
I think it makes sense with the new `format` approach. My only concern if we do this is the parity with the `RelativeTime` component name.